AutonomousDemo/DeterminePath.cs

```diff
@@ -19,24 +19,24 @@ public Velocity CalculateDriveVelocity(Vehicle vehicle)
             return vehicle.VehicleMotion.CurrentVelocity;
         }
 
-        public Velocity StoppingDistance(Vehicle vehicle, double destinationDistance)
-        {
-            var stoppingDistance = destinationDistance - vehicle.VehicleMotion.DistanceTraveled;
-            if (stoppingDistance < vehicle.VehicleMotion.CurrentVelocity.Speed
-                || stoppingDistance >= vehicle.VehicleMotion.CurrentVelocity.Speed - 10)
-            {
-                //slow down faster
-                var slowerVelocity = vehicle.VehicleMotion.Decelerate(vehicle.VehicleMotion.CurrentVelocity);
-                vehicle.VehicleMotion.Decelerate(slowerVelocity);
-            }
-            //need a range maybe
-            else if (stoppingDistance == vehicle.VehicleMotion.CurrentVelocity.Speed
-                || stoppingDistance > vehicle.VehicleMotion.CurrentVelocity.Speed && stoppingDistance <= vehicle.VehicleMotion.CurrentVelocity.Speed + 10) //10m/s
-            {
-                return vehicle.VehicleMotion.Decelerate(vehicle.VehicleMotion.CurrentVelocity);
-            }
+        //public Velocity StoppingDistance(Vehicle vehicle, double destinationDistance)
```

Comment:
Probably shouldn't bother checking in a bunch of commented code. You can always get the old version later if you delete it anyway.